packages/preview/typso/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "typso"
```

Comment:
I'm not sure about the package name, it is generally discouraged to use "typst" or something close to it, and in this case I can't really see the relation with the package purpose (is it just a mix of **Typs**t and bing**o**?).